src/sagemaker/cli/compatibility/v2/ast_transformer.py

```diff
@@ -52,52 +62,85 @@ class ASTTransformer(ast.NodeTransformer):
     """
 
     def visit_Call(self, node):
-        """Visits an ``ast.Call`` node and returns a modified node, if needed.
+        """Visits an ``ast.Call`` node and returns a modified node or None.
         See https://docs.python.org/3/library/ast.html#ast.NodeTransformer.
 
         Args:
             node (ast.Call): a node that represents a function call.
 
         Returns:
-            ast.Call: a node that represents a function call, which has
-                potentially been modified from the original input.
+            ast.AST: if the returned node is None, the original node is removed
+            from its location. Otherwise, the original node is replaced with the
+            returned node.
```

Comment:
use a hanging indent for argument/return value descriptions (applies throughout)

```suggestion
            ast.AST: if the returned node is None, the original node is removed
                from its location. Otherwise, the original node is replaced with the
                returned node.
```

Reply:
Fixed